www/routes/std.tsx

```diff
@@ -0,0 +1,12 @@
+import { Handlers, RouteConfig } from "$fresh/server.ts";
+import { serveFile } from "https://deno.land/std@0.176.0/http/file_server.ts";
+
+export const handler: Handlers<unknown> = {
+  async GET(req, cxt) {
+    let { params: { path } } = cxt;
+    let location = new URL(`../../stdlib/${path}`, import.meta.url)
```

Comment:
do we need to sanitize this input?

Reply:
I don’t think so. Fresh normalizes all paths, and deno deploy forbids reads outside the source… but I will add a test.

Reply:
done.